### PR TITLE
feat: added support for separate PRs for playlist videos

### LINKED ISSUE #65: create separate PRs for each video in the playlist

### PR DESCRIPTION
**Pull request by masterchief164**
This PR fixes #65

### CODE REVIEW

File: transcriber.py [hunk 1]
```diff
@@ -63,12 +62,12 @@ def add(
...
     filename = application.process_source(source=source, title=title, event_date=event_date, tags=tags,
                                           category=category, speakers=speakers, loc=loc, model=model, username=username,
-                                          curr_time=curr_time, source_type=source_type, created_files=created_files,
+                                          source_type=source_type, created_files=created_files,
                                           chapters=chapters)
     if filename:
         """ INITIALIZE GIT AND OPEN A PR"""
-        branch_name = loc.replace("/", "-")
-        subprocess.call(['bash', 'github.sh', branch_name, username, curr_time, filename[:-4]])
+        # branch_name = loc.replace("/", "-")
```
[the remark this answers is not in the document]
Comment (masterchief164) [reply transcriber.py:69]: Yeah I'll remove them.

**Comment by adamjonas** [comment]
This didn't run even though a playlist was detected
Screen Shot 2023-02-17 at 4 58 46 PM

This ran but didn't run the others in the list
Screen Shot 2023-02-17 at 4 59 30 PM

**Comment by masterchief164** [comment]
> This didn't run even though a playlist was detected Screen Shot 2023-02-17 at 4 58 46 PM
> 
> This ran but didn't run the others in the list Screen Shot 2023-02-17 at 4 59 30 PM

The second part is running as expected as a single video will be given preference over a playlist. The first issue was a bug in the application which has been resolved in the update pr #64.

**Comment by adamjonas** [comment]
> single video will be given preference over a playlist

But the playlist is a superset of the single video. Why not do both?

**Comment by masterchief164** [comment]
Alright I'll give preference to playlists over a single video. I guess that should solve it.

**Comment by adamjonas** [comment]
Still problems on dfe20cee18ce2ed09aee9d96d406db420cd6c2fd which produced https://github.com/adamjonas/bitcointranscripts/pull/65, https://github.com/adamjonas/bitcointranscripts/pull/64, https://github.com/adamjonas/bitcointranscripts/pull/63, https://github.com/adamjonas/bitcointranscripts/pull/62

- the commit message is showing my local file path
- there is no way to differentitate the PRs based on the PR description or message. The title of the video needs to find it's way in somehow.
- Every PR creates it's own index page. I don't know how to solve for that, but given these are separate PRs but that's a problem and possibly a showstopper.

Screen Shot 2023-02-27 at 1 42 54 PM
Screen Shot 2023-02-27 at 1 42 41 PM
Screen Shot 2023-02-27 at 1 42 33 PM
Screen Shot 2023-02-27 at 1 42 21 PM
Screen Shot 2023-02-27 at 1 42 13 PM

**Comment by masterchief164** [comment]
> Still problems on [dfe20ce](https://github.com/bitcointranscripts/tbtbtc/commit/dfe20cee18ce2ed09aee9d96d406db420cd6c2fd) which produced [adamjonas/bitcointranscripts#65](https://github.com/adamjonas/bitcointranscripts/pull/65), [adamjonas/bitcointranscripts#64](https://github.com/adamjonas/bitcointranscripts/pull/64), [adamjonas/bitcointranscripts#63](https://github.com/adamjonas/bitcointranscripts/pull/63), [adamjonas/bitcointranscripts#62](https://github.com/adamjonas/bitcointranscripts/pull/62)
> 
> * the commit message is showing my local file path
> * there is no way to differentitate the PRs based on the PR description or message. The title of the video needs to find it's way in somehow.
> * Every PR creates it's own index page. I don't know how to solve for that, but given these are separate PRs but that's a problem and possibly a showstopper.
> 
> Screen Shot 2023-02-27 at 1 42 54 PM
> 
> Screen Shot 2023-02-27 at 1 42 41 PM
> 
> Screen Shot 2023-02-27 at 1 42 33 PM
> 
> Screen Shot 2023-02-27 at 1 42 21 PM
> 
> Screen Shot 2023-02-27 at 1 42 13 PM

The 1st issue has been resolved.
The 3rd one is not an issue at all, as soon as any of the PRs are merged git internally handles the creation of the new index pages and doesn't make a change in the files
For the 2nd issue we can change the body(message) to `transcribed to bitcoin transcript for <file name>` from `transcribed to bitcoin transcript for <directory>`.

**Comment by adamjonas** [comment]
still greeted by this for PRs from a playlist but let's make it a follow-up:

Screen Shot 2023-03-03 at 4 42 51 PM

**Comment by masterchief164** [comment]
This PR added the video name to the PR body as a comment rather than changing the PR title. We can change that at a later point though.

**Pull request merged**